modules/uc/craco.config.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2021 Terminus, Inc.
```

Comment:
remove yarn.lock & package-lock files if it's part of erda ui monorepo. and add this module to lerna config